resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketForm/useCreateAchievementTicketForm.ts

```diff
@@ -0,0 +1,96 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { toastMessage } from '@/common/components/+vendor/BaseToaster';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { TicketType } from '@/common/utils/generatedAppConstants';
+
+const createAchievementTicketFormSchema = z.object({
+  /** @see TicketType.php */
+  issue: z.enum(['DidNotTrigger', 'TriggeredAtWrongTime', 'NetworkIssue']),
+  emulator: z.string().min(1),
+  emulatorVersion: z.string().optional(),
+  core: z.string().optional(),
+  mode: z.enum(['hardcore', 'softcore']),
+  hash: z.string().min(1),
+  description: z
+    .string()
+    .min(25, { message: 'Please be more detailed in your description.' })
+    .refine((desc) => !/(n'?t|not?).*(work|trigger)/i.test(desc), {
+      message: 'Please be more detailed in your description.',
+    })
+    .refine((desc) => !/(internet|wifi|connection|network)/i.test(desc), {
+      message: 'Do not open tickets for network issues.',
+    }),
```

Comment:
This appears to completely prevent submission. There are times where "connection" or "network" might be appropriate to include in the description.

> I can't seem to unlock the connection between John and Jane in my social network.

Reply:
In latest, I believe I've updated the form validation / info message logic to mirror what Alpine.js is doing.

The form should only be invalidated by the description now if the user enters unhelpful text that's less than 25 characters long.